### DISCUSSION TITLE
Is this repo still active?

### DISCUSSION THREAD

**Discussion by artyPD** [AIRLegend/aitrack#259] [General]
I don't mean to barge in and make demands. However, I'm just interested to know if this repo is still being worked on?

I am a big fan of the project and benefit from it immensely. Great job to all of you! Hopefully there will be some releases again soon.

Have a great day!

**Comment by tessierp** [comment]
Unfortunately, I do not think it is. There has been Pull Requests for a bit now without any actions.

**Reply by artyPD** [reply]
That's unfortunate. It's a fantastic tool for those on a budget. Thanks for the reply.

**Reply by tessierp** [reply]
Definitely is a great tool. And it really rivals the high-end stuff I find. There are a few issues such as, it is a bit jumpy in low light. Hopefully someone who seems some interest picks up the project where it left off.

**Comment by artyPD** [comment]
Forgot to close this.
